int.satyg

```diff
@@ -26,6 +26,12 @@ module Int : sig
   val sup : int list -> int
   val inf : int list -> int
 
+  val (<) : int -> int -> bool
+  val (>) : int -> int -> bool
+  val (>=) : int -> int -> bool
+  val (<=) : int -> int -> bool
+  val (==) : int -> int -> bool
```

Comment:
I needed these in the Float module because Float defines comparison operators for float and shadows that for int.